lib/numtheor.gi

```diff
@@ -1385,24 +1391,16 @@ end );
 
 
 InstallGlobalFunction(PValuation,function(n,p)
-local a,v;
-  if not IsPrimeInt(p) or not IsRat(n) then
+local v;
+  if not IsPosInt(p) or not IsRat(n) then
```

Comment:
Hmm... I change `IsPrimeInt` to `IsPosInt` as that avoided a pointless and potentially costly primality test. However, this meant it used to accept negative values for `p`, too. So perhaps I should change `IsPosInt(p)` to `IsInt(p) and p <> 0` ?
Might just as well insert a better error message too... And finally, is there any reason this very useful function is not documented???